datumaro/components/progress_reporting.py

```diff
@@ -1,12 +1,15 @@
-# Copyright (C) 2022 Intel Corporation
+# Copyright (C) 2023 Intel Corporation
 #
 # SPDX-License-Identifier: MIT
 
 from __future__ import annotations
 
 import math
+import time
 from typing import Iterable, Optional, Tuple, TypeVar
 
+from tqdm import tqdm
```

Comment:
```suggestion
from tqdm.auto import tqdm
```

Reply:
Fixed in https://github.com/openvinotoolkit/datumaro/pull/927/commits/fed66ea98909b7a54af83c66ff50349640793361